src/Controller/Controller.php

```diff
@@ -159,7 +163,7 @@ class Controller implements EventListenerInterface, EventDispatcherInterface
      *
      * @var bool
      */
-    public $autoRender = true;
+    protected $autoRender = true;
```

Comment:
I think this change is meaningless. Because when I touch this property, I am in the action of the controller usually. So I can continue to use this property without any warnings even if I am using some great IDE. Probably, it's the time to allow us to use `private` modifiers. To be honest, I think clean APIs wouldn't prefer setters/getters for public/protected properties or getters for private mutable objects.